db/db_impl/db_impl_open.cc

```diff
@@ -1930,21 +1930,24 @@ Status DBImpl::Open(const DBOptions& db_options, const std::string& dbname,
     // vast majority of all files), we'll pass the size to SstFileManager.
     // For all other files SstFileManager will query the size from filesystem.
 
-    std::vector<LiveFileMetaData> metadata;
-
```

Comment:
We don't need to use `impl->mutex` since it already been used in `GetAllColumnFamilyMetaData`.